dZoneManager/Zone.cpp

```diff
@@ -42,6 +42,12 @@ void Zone::LoadZoneIntoMemory() {
 	if (m_ZoneFilePath == "ERR") return;
 
 	AssetMemoryBuffer buffer = Game::assetManager->GetFileAsBuffer(m_ZoneFilePath.c_str());
+
+	if (!buffer.m_Success) {
+		Game::logger->Log("Zone", "Failed to load %s", m_ZoneFilePath.c_str());
+		return;
+	}
```

Comment:
I'm not sure how I feel about letting the server continue running if the zone is unable to be loaded.  What are your thoughts on this throwing an exception instead since you kind of need the whole zone in order to play on the server.

This comment also applies to the lvl triggers below.  If we threw an exception at the very least we would have a known spot the exception would be located at.

Reply:
Agreed, this should throw an error and say "Cannot Find File" 
Should also update the triggers message to say the same, but not throw an error